qsirecon/cli/parser.py

```diff
@@ -475,16 +475,36 @@ def parse_args(args=None, namespace=None):
     work_dir.mkdir(exist_ok=True, parents=True)
 
     # Run ingression if necessary
-    if config.workflow.input_type in ("ukbb", "hcpya"):
-        from ingress2qsirecon.utils.workflows import ingress2qsirecon
+    if config.workflow.input_type in ("hcpya", "ukb"):
+        import os.path as op
+        import shutil
 
+        import ingress2qsirecon
+        from ingress2qsirecon.utils.functions import create_layout
+        from ingress2qsirecon.utils.workflows import create_ingress2qsirecon_wf
+
+        # Fake BIDS directory to be created
         config.execution.bids_dir = work_dir / "bids"
 
-        wf = ingress2qsirecon(
-            config.execution.input_dir,
-            config.execution.bids_dir,
-            config.workflow.input_type,
+        # Make fake BIDS files
+        ingress2recon_dir = op.dirname(ingress2qsirecon.__file__)
```

Comment:
Maybe use `nipreps/acres` (https://github.com/nipreps/acres) for loading package data.

Reply:
Done in latest commit:

```
        from ingress2qsirecon.data import load_resource

        # Make fake BIDS files
        bids_scaffold = str(load_resource("bids_scaffold/"))
        if not op.exists(op.join(config.execution.bids_dir, "dataset_description.json")):
            shutil.copytree(
                bids_scaffold,
                config.execution.bids_dir,
                dirs_exist_ok=True,
            )
```